### DISCUSSION TITLE
ImplicitPlaneWidget in trame/paraview

### DISCUSSION THREAD

**Discussion by vminedev** [Kitware/trame#169] [Q&A]
I'm trying to interactively define the clipping plane for an unstructuredgrid like Show plane checkbox in Paraview:

![Show plane](https://user-images.githubusercontent.com/117809455/208097496-2a271d98-45a4-48fa-a3e3-e3097df1f037.png)

I have found a solution with vtk:
https://kitware.github.io/vtk-examples/site/Python/Widgets/ImplicitPlaneWidget2

Is it possible to define clipping plane with simple package of Paraview(pvpython)?

**Comment by jourdain** [comment]
It is, but it currently requires deep ParaView knowledge. To bridge that gap, we are going to build a helper to expose those widgets simply for both VTK and ParaView so we can keep that easy-to-use trame vibe.

Unfortunately, I don't have a timeframe for you, but if you want to make it a priority, feel free to reach out to [Kitware for support](https://www.kitware.com/contact/).
